shared/actions/config/index.js

```diff
@@ -33,6 +34,79 @@ function getConfig (): AsyncAction {
   }
 }
 
+let followers = {}
```

Comment:
This seems like it could be a maintenance gotcha later we're now storing two copies of the data, one copy outside of the typical Flux pattern.